src/CMakeLists.txt

```diff
@@ -17,11 +17,12 @@ add_subdirectory(third_party)
 add_subdirectory(metrics)
 add_subdirectory(value_generators)
 add_subdirectory(gennylib)
+add_subdirectory(testlib)
 add_subdirectory(cast_core)
 add_subdirectory(driver)
 
 # This makes the project importable from the build directory
-export(TARGETS value_generators metrics gennylib gennylib_cast gennylib_core loki cast_core
+export(TARGETS value_generators metrics gennylib loki cast_core
```

Comment:
You should be able to replace this line with `export(EXPORT GennyLibraryConfig` now that all the targets are being installed to `GennyLibraryConfig`

Reply:
⬆️ 

Alternatively, could we rename the `GennyLibraryConfig` export-name to something else if we don't want it to be the content of `GennyLibraryConfig.cmake`?

Reply:
> Alternatively, could we rename the GennyLibraryConfig export-name to something else if we don't want it to be the content of GennyLibraryConfig.cmake?

I tried my best to just kill `GennyLibraryConfig` but it was failing to compile on linux variants without it (but was working on my laptop). So I just threw my hands up. I like your idea but I can't bear to spend another test iteration on this so just gonna leave it 😔